packages/neos-ui-i18n/src/index.js

```diff
@@ -51,8 +51,44 @@ export default class I18n extends Component {
         return [packageKey, sourceName, id];
     }
 
+    substitutePlaceholders(textWithPlaceholders, parameters) {
+        let startOfPlaceholder;
+        while ((startOfPlaceholder = textWithPlaceholders.indexOf('{')) !== -1) {
```

Comment:
Not sure about this `while` loop in the render call, can we solve this in another way maybe? Also it would be nice to explain/document this magic a bit if a newcomer stumbles upon it